.github/workflows/nexus-publish-osgi.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will build a package using Maven and then publish it to
+# qbic-repo.qbic.uni-tuebingen.de packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Nexus Package
+
+on:
+  push:
+    branches:
+      - development
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```

Comment:
here its still java 11, is that intended?

Reply:
@sven1103 I think it is?

Reply:
wont work, as data model lib is Java 8